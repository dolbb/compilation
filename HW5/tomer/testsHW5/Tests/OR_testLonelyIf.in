int Int(){print("set!\n");return 500;}
void WasHere(){print("was here\n");}
void main(){
int target = 0;
if(true) int t6;
if(true) int t6 = Int();
if(true) target = Int();
printi(target);
print("\n");target = 0;
if(true)WasHere();if(true){
int t6 = Int();
printi(t6);
}
print("\n");if(true){
int t6;
t6 = Int();
printi(t6);
}
print("\n");if(true) {target = Int();}
printi(target);
print("\n");
target = 0;
if(true) {WasHere();}
if(true) if (true) WasHere();
if(true) if (true) WasHere(); 
	else int x;
if(true) if (false) int x; 
	else WasHere();
if(true)while (true) {WasHere();break;}
if(true)switch (target) {
case 0: WasHere();break;
case 1b: int x;break;
};
if(true)while (target == 0) target = Int();
}
