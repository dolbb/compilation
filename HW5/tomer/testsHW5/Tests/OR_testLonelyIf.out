Loaded: ./exceptions.s
set!
set!
500
was here
set!
500
set!
500
set!
500
was here
was here
was here
was here
was here
was here
set!
